integration/src/main/java/scripts/testscripts/PrivateControlledAiNotebookInstanceLifecycle.java

```diff
@@ -62,9 +63,10 @@ protected void doSetup(List<TestUserSpecification> testUsers, WorkspaceApi works
   }
 
   @Override
+  @SuppressFBWarnings(value = "DLS_DEAD_LOCAL_STORE")
   protected void doUserJourney(TestUserSpecification testUser, WorkspaceApi workspaceApi)
       throws Exception {
-    String projectId = CloudContextMaker.createGcpCloudContext(getWorkspaceId(), workspaceApi);
+    String unused = CloudContextMaker.createGcpCloudContext(getWorkspaceId(), workspaceApi);
```

Comment:
nit: if we aren't using this, it's probably better to just ignore the return value of this call instead